xlive/Blam/Engine/camera/observer.h

```diff
@@ -4,6 +4,8 @@
 
 /* constants */
 
+constexpr real_point2d k_observer_default_cross_hair_position{ {0.f, 0.138f} };
```

Comment:
This should be a constant instead of constexpr

Reply:
constantexpr in this context is the same as constant.

Reply:
No, it's duplicated in 2 TUs right now, keep it as a constant assigned in the cpp file and have it as an extern in the header